src/compiler/scala/tools/nsc/symtab/classfile/ClassfileParser.scala

```diff
@@ -1184,11 +1210,65 @@ abstract class ClassfileParser(reader: ReusableInstance[ReusableDataReader]) {
       AnyRefClass // Force scala.AnyRef, otherwise we get "error: Symbol AnyRef is missing from the classpath"
       assert(bytes != null, s"No Scala(Long)Signature annotation in classfile with ScalaSignature attribute: $clazz")
       unpickler.unpickle(bytes, 0, clazz, staticModule, file.name)
+    } else if (isTASTY) {
+
+      def parseTASTYFile(): Array[Byte] = file.underlyingSource match { // TODO: simplify when #3552 is fixed
```

Comment:
```suggestion
      def parseTASTYFile(): Array[Byte] = file.underlyingSource match { // TODO: simplify when lampepfl/dotty#3552 is fixed
```

in case that comment applies here